scales/definitions/cylinder_mm_d370_h330_1-30l_i1.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "./schemas/scale-config",
+    "scale-config": {
+        "containerForm": "cylinder",
+        "lengthUnit": "mm",
+        "volumeUnit": "l",
+        "diameter": 370,
+        "height": 330,
+        "minVolume": 2,
+        "maxVolume": 30,
+        "output": "./scales/svgs/{0}.svg",
+        "description": "Sansone Edelstahlbehälter 18/10 30 Liter",
```

Comment:
Can you please use the more generic description text (as in other scales)? You can place the additional information in the `successfullyAppliedTo` field.